test/jdk/tools/launcher/I18NArgTest.java

```diff
@@ -97,12 +97,21 @@ static void execTest(String unicodeStr, String hexValue) {
 
         // Test via JDK_JAVA_OPTIONS
         Map<String, String> env = new HashMap<>();
-        String cmd = "-Dtest.src=" + TEST_SOURCES_DIR.getAbsolutePath() +
-                " -Dtest.classes=" + TEST_CLASSES_DIR.getAbsolutePath() +
-                " -cp " + TEST_CLASSES_DIR.getAbsolutePath() +
-                " I18NArgTest " + unicodeStr + " " + hexValue;
-        env.put("JDK_JAVA_OPTIONS", cmd);
-        tr = doExec(env, javaCmd);
+        String sysPropName = "foo.bar";
+        // When pass "-Dfoo.bar=<unicodestr>" via the JDK_JAVA_OPTIONS environment variable,
+        // we expect that system property value to be passed along to the main method with the
+        // correct encoding
+        // If <unicodestr> contains space or tab, it should beenclosed with double quotes.
```

Comment:
Nit: space is missing between "be" and "enclosed"

Reply:
Thank you for your comment, I fixed it.